variables.tf

```diff
@@ -321,10 +321,10 @@ variable "metrics_router_routes" {
   type = list(object({
     name = string
     rules = list(object({
-      action = string
-      targets = list(object({
+      action = optional(string, "send")
+      targets = optional(list(object({
```

Comment:
Is the right? Won't you always need a target to create a route?

Reply:
Yes, if the action is "drop" then user doesn't need to add a target. 

<img width="1327" alt="Screenshot 2024-12-18 at 4 28 53 PM" src="https://github.com/user-attachments/assets/d211af79-9930-49f4-92a5-09597faff62b" />

Reply:
ok, thanks for clarifying